src/qibolab/instruments/qblox/port.py

```diff
@@ -33,12 +33,10 @@ class QbloxOutputPort(Port):
     """qibolab.instruments.port.Port interface implementation for Qblox
     instruments."""
 
-    def __init__(
-        self, module, sequencer_number: int, port_number: int, port_name: str = None
-    ):
+    def __init__(self, module, port_number: int, port_name: str = None):
         self.name = port_name
         self.module = module
-        self.sequencer_number: int = sequencer_number
+        self.sequencer_number: int = port_number
         self.port_number: int = port_number
```

Comment:
If these are now the same, do we need both of them? Is there a place where they are modified later on?

Reply:
I remember discarding one of them and the driver were not working properly although if I can't find any modification of this attributes in the rest of the code. For now I would keep this, since I can't test on hardware, and if we merge I will open an issue about it.

Reply:
Issue to be opened

Reply:
Solved in https://github.com/qiboteam/qibolab/issues/742